Makefile

```diff
@@ -306,19 +310,18 @@ test: test/go test/lint
 test/go: cnquery/generate test/go/plain
 
 test/go/plain:
-	# TODO /motor/docker/docker_engine cannot be executed inside of docker
-	go test -cover $(shell go list ./... | grep -v '/motor/discovery/docker_engine')
+	go test -cover $(shell go list ./... | grep -v '/providers/slack')
 
 test/go/plain-ci: prep/tools providers/build
-	gotestsum --junitfile report.xml --format pkgname -- -cover $(shell go list ./... | grep -v '/vendor/' | grep -v '/motor/discovery/docker_engine')
+	gotestsum --junitfile report.xml --format pkgname -- -cover $(shell go list ./... | grep -v '/vendor/' | grep -v '/providers/slack')
```

Comment:
this excludes providers from global testing, need to come up with something that does testing for each provider